samples/connectedhomeip/lock/src/gen/call-command-handler.cpp

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
I wonder if we can have such license within our SDK. I would contact Carles and ask.